src/triangulation.rs

```diff
@@ -322,6 +322,15 @@ pub trait Triangulation: Default {
         self.s().fixed_vertices()
     }
 
+    /// Get a vertex by its index
+    fn get_vertex(
+        &self,
+        index: usize,
+    ) -> Option<VertexHandle<Self::Vertex, Self::DirectedEdge, Self::UndirectedEdge, Self::Face>>
```

Comment:
I'm afraid that we need to add a `#[allow(clippy::type_complexity)]` to appease clippy in this instance...